src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const minDiscount = 20;
+  const maxDiscount = 50;
```

Comment:
I suggest to move these constants out of `calculateRentalCost`
```suggestion
  const MIN_DISCOUNT= 20;
  const MAX_DISCOUNT = 50;
```